packages/flutter/lib/src/material/carousel.dart

```diff
@@ -335,6 +438,18 @@ class CarouselView extends StatefulWidget {
   /// The child widgets for the carousel.
   final List<Widget> children;
 
+  /// Called to build carousel item on demand.
+  ///
+  /// Will be called only for indices greater than or equal to zero and less than [itemCount].
+  ///
+  /// Must return a non-null widget for valid indices.
+  final Widget Function(BuildContext, int)? itemBuilder;
```

Comment:
Maybe create a `typedef CarouselItemBuilder = Widget Function(BuildContext, int)?` so here would look simper:) typedef should be placed on the top of this file.